asciidoc/client-applications.adoc

```diff
@@ -182,15 +148,49 @@ include::{javascript-examples}/examples.test.js[tags=config-custom-resolver]
 ----
 ======
 
-[.include-with-python]
-======
-[source, python]
-----
-include::{python-examples}/custom_resolver_example.py[tags=custom-resolver]
-----
-======
+// [.include-with-python]
+// ======
+// [source, python]
+// ----
+// include::{python-examples}/custom_resolver_example.py[tags=custom-resolver]
+// ----
+// ======
+====
+
+=== Routing table
+
+The routing table is the glue between the driver connectivity layer and the database surface.
+This table contains a list of server addresses, grouped as readers and writers, and is refreshed automatically by the driver as required.
+
+The driver does not expose any API to work directly with the routing table, but it can sometimes be useful to explore when troubleshooting a system.
+
+[[driver-routing-context]]
+=== Routing context
+
+A routing context can be included as the query part of a `neo4j:// URI`. Routing contexts are defined by means of server policies and allow customisation of the contents of the routing table.
```

Comment:
American English customisation -> customization

Reply:
What's that, old boy? Tally ho, pip-pip.

Reply:
It's fixed y'all!